### ISSUE TITLE
Can we add 'pool members' option in bigip_gtm_pool.py

### ISSUE THREAD

**Issue by payalsin** [F5Networks/f5-ansible#140]
Can we add an option to add 'pool members' to the bigip_gtm_pool module.

**Comment by insidepacket** [comment]
Yes. We need this feature!

**Comment by caphrim007** [comment]
yeah we can add it. this module requires a REST refactor though before this can be added. I am almost finished with a refactor.

**Comment by activeactive** [comment]
I need ability to enable/disable GTM pool members. Will it be in this module down the road?

**Comment by caphrim007** [comment]
@activeactive that ability would be part of a bigip_pool_member module.

**Comment by activeactive** [comment]
@caphrim007 I don't see bigip_gtm_pool_member module. Is it in planning stage?

**Comment by caphrim007** [comment]
@activeactive correct, we would need to add it. There is not yet an issue for it. Would you be able to add an issue?

**Comment by activeactive** [comment]
Just added the issue.
https://github.com/F5Networks/f5-ansible/issues/286

**Comment by delgadillo22** [comment]
Looks like Fidelity requirement

**Comment by WRJFontenot** [comment]
Must have.
